src/ue/comments/comments.controller.ts

```diff
@@ -28,9 +31,13 @@ export class CommentsController {
     ERROR_CODE.NO_SUCH_UE,
     'This error is sent back when there is no UE associated with the code provided.',
   )
-  async getUEComments(@GetUser() user: User, @Query() dto: GetUeCommentsReqDto): Promise<Pagination<UeCommentResDto>> {
+  async getUEComments(
```

Comment:
On peut aussi le décorateur non ? Sauf si on le fait dans un deuxième temps (ainsi que partout où cela s'applique)
```suggestion
  @RequireApiPermission(API_SEE_OPINIONS_UE)
  async getUEComments(
```

Reply:
Pourquoi ? On veut que les UEs soient publiques

Reply:
Edit: oh nan j'suis stupide, ça faisait 5mins que je lisais la ligne je pensais que ça retournait l'ue